core/node/state_keeper/src/keeper.rs

```diff
@@ -582,47 +651,70 @@ impl ZkSyncStateKeeper {
                     "L1 batch #{} should be sealed unconditionally as per sealing rules",
                     updates_manager.l1_batch.number
                 );
+
+                // Push the current block if it has not been done yet
+                if is_last_block_sealed {
+                    let new_l2_block_params = self
+                        .wait_for_closing_l2_block_params_when_closing_batch(
```

Comment:
Trick to make pass the unit tests.

In case of [OpenBlock, SealBatch] , if we replace this line with `wait_for_new_l2_block_params`, unit tests will fail because `wait_for_new_l2_block_params` is called twice